xicam/gui/windows/mainwindow.py

```diff
@@ -162,10 +165,14 @@ def open(self, header):
         if self.currentGUIPlugin is None:
             msg.notifyMessage("Please select a gui plugin from the top before trying to open an image.")
             return
+        append = False
```

Comment:
hold off on these changes until we can discuss integrating the ensembles/workspace concept into the larger application

Reply:
Removed these changes.